app/views/peeps.erb

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+    <title> Chitter </title>
+  </head>
+
+  <div>
+      <h1> Latest Peeps </h1>
+
+    <% if @current_user %>
```

Comment:
Rather than a view which had two different states, I would expect this to be two different views (and possibly two different routes)